### ISSUE TITLE
response['message'] could not get the returned data

### ISSUE THREAD

**Issue by neypro** [terry3041/pyChatGPT#147]
Today, I use pyChatGPT and realized response['message'] could not get the returned data. Is the source code changed?

**Comment by Aionstn** [comment]
File "D:\aconda\..\pyChatGPT\pyChatGPT.py", line 460, in send_message
    conversation_id = matches.group()
AttributeError: 'NoneType' object has no attribute 'group'

It looks like something went wrong with the regex

**Comment by neypro** [comment]
Have you fixed this error yet? Thanks

**Comment by mrryan2** [comment]
I also got the same error. Is anyone working on this? Do you want help ?

**Comment by mrryan2** [comment]
Any news on a fix for this problem ?

**Comment by Aionstn** [comment]
It is possible that chatgtp deleted all the returned information after detecting selenium

**Comment by mrryan2** [comment]
I dont think so, but whatever it is i hope it gets resolved pretty soon. I have a large project that this import is perfect for :(

**Comment by mrryan2** [comment]
Still no  solution for this ?

**Comment by suspiciousRaccoon** [comment]
getting the same error : /

**Comment by mrryan2** [comment]
I havent fixed it but i have found a way around. If you remove your breakpoints (VSC) and dont use the response object inside the python script you can use 'Super Power ChatGPT' Chrome extension to download all the conversations. Depending on your project that may or may not be useful, but it works for me.
I would still like to find a solution though so i will keep looking and if i find something i will post it here.
